WTWSMS/history/characters/latin.txt

```diff
@@ -0,0 +1,12 @@
+1310014 = {
+   name = "Sissinius" #Sissinius 2 PLRE III
+	religion = "nicene"
+	culture = "latin"
```

Comment:
Why is this character Latin?

Reply:
Rome had already fell and the most powerful character is Latin too.

Reply:
Latin is getting removed anyway.

Reply:
Oh, What a pity, didn´t know that.

Reply:
What do you mean by pity?

Reply:
Solved

Reply:
I liked the concept of the Latin culture

Reply:
I´m answering the Cilicia thing here as I cannot reply your comment. Yes, it has been solved.